docs/content/configuration/csi_ug_config_create_pvc.md

```diff
@@ -25,87 +25,83 @@ Use the following sections, according to your PVC needs:
 
```

Comment:
this may not be related to the changes here, I can't even comment on the relevant line (line #17), but in general, lines of the form:
`The <k8s-object-type>/<filename> created message is emitted.` should be of the form:
`The <k8s-object-type>/<k8s-object-name> created message is emitted.`, because the k8s server never knows the file name.

e.g.:
`The persistentvolumeclaim/<filename> created message is emitted.`, should be:
`The persistentvolumeclaim/<pvc-name> created message is emitted.`

Reply:
DONE